chapter18_questions

```diff
@@ -0,0 +1,34 @@
+Anthony Wallace
+## chapter 18 notes
+
+#### chapter 18 Questions
+if an action is only moral when can be back up by reason, what happen when one person reason for doing something is different then another person reals for doing something which reason do you chose?
+
```

Comment:
When (it) can be ... What happen(s) when one person(s) reason ...  another (person reals?) for doing something (which reason do you chose?)